src/index.js

```diff
@@ -1,18 +1,16 @@
 import React from "react";
-import ReactDOM from "react-dom";
+import { createRoot } from 'react-dom/client';
```

Comment:
I was seeing this warning in the console: `Warning: You are importing createRoot from "react-dom" which is not supported. You should instead import it from "react-dom/client"`

Learned this from https://stackoverflow.com/questions/71743492/issue-importing-createroot-from-react-dom-client